packages/trader/src/Modules/Trading/Components/Elements/Multiplier/__tests__/cancel-deal-mobile.spec.jsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import { render, screen } from '@testing-library/react';
+import { mockStore } from '@deriv/stores';
+import TraderProviders from '../../../../../../trader-providers';
+import CancelDeal from '../cancel-deal-mobile';
+import * as multiplier_functions from 'Stores/Modules/Trading/Helpers/multiplier';
+
+const default_mocked_props = {
+    has_cancellation: false,
+    has_take_profit: false,
+    has_stop_loss: false,
+    onChangeMultiple: jest.fn(),
+    cancellation_duration: '60m',
+};
+const default_mock_store = {
+    modules: {
+        trade: {
+            cancellation_range_list: [],
+        },
+    },
+};
+
+const mockOnToggleCancellation = jest.spyOn(multiplier_functions, 'onToggleCancellation');
+
+jest.mock('@deriv/components', () => ({
+    ...jest.requireActual('@deriv/components'),
+    Dialog: ({ children, ...props }) => (
+        <div style={{ display: props.is_visible ? 'block' : 'none' }}>
+            {children}
+            <button onClick={props.onCancel}>Cancell button</button>
+            <button onClick={props.onConfirm}>Confirm button</button>
+        </div>
+    ),
+}));
+
+describe('<CancelDeal />', () => {
+    const mockCancelDeal = (mocked_store, mocked_props) => {
+        return (
+            <TraderProviders store={mocked_store}>
+                <CancelDeal {...mocked_props} />
+            </TraderProviders>
+        );
+    };
+
+    it('should render the component with checkbox and popover', () => {
+        const mock_root_store = mockStore(default_mock_store);
+        render(mockCancelDeal(mock_root_store, default_mocked_props));
+
+        const info_icon = screen.getByTestId(/dt_popover_wrapper/i);
+        userEvent.hover(info_icon);
+
+        expect(screen.getByText('Deal cancellation')).toBeInTheDocument();
```

Comment:
`'Deal cancellation'` is also a part of another phrase, so here I use string instead of reg exp